lint.sh

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env bash
 set -e
 
+# ℹ️ Linting Dockerfiles (**/Dockerfile)
+# https://github.com/hadolint/hadolint
+docker run --rm -v "$(pwd)":/code -v "$(pwd)/.hadolint.yml":/.hadolint.yml hadolint/hadolint:latest-alpine sh -c "find . -name Dockerfile -print -o -type d -name node_modules -prune | xargs hadolint"
```

Comment:
No longer tries to lint `Dockerfile`s installed in node dependencies (i.e. ignores `node_modules`)